### PR TITLE
Added features part

### PR DESCRIPTION
**Pull request by lemunozm**
Added main features part to the first page of documentation.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+and enables them to take part as DDS publishers and/or subscribers in the DDS Global Data Space.
 
 *Micro XRCE-DDS* provides both, a plug and play *Micro XRCE-DDS Agent* and an API layer which allows you to implement your *Micro XRCE-DDS Clients*.
 
 .. image:: images/xrcedds_architecture.svg
 
+Main features
+~~~~~~~~~~~~~
+High performance.
+    Uses a static low level serialization library `(MicroCDR) <https://github.com/eProsima/Micro-CDR>`_ that serialize in `XCDR <https://www.omg.org/spec/DDS-XTypes/1.2/PDF>`_.
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
    Uses a static low-level serialization library `(MicroCDR) <https://github.com/eProsima/Micro-CDR>`_ that serialize in `XCDR <https://www.omg.org/spec/DDS-XTypes/1.2/PDF>`_.
```

File: docs/index.rst [hunk 2]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+
+Low resouces.
+    The client library is dynamic memory free and static memory free.
+    This means that the only memory charge is due by the stack growth.
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
    This means that the only memory charge is due to the stack growth.
```

File: docs/index.rst [hunk 3]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+    It can manage a simple publisher/subscriber with less of 2KB of RAM.
+    Besides, the client is built with a *profiles* concept, that allows to add or remove functionality to the library in order to reduce the binary size.
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
    Besides, the client is built with a *profiles* concept, that allows to add or remove functionality to the library changing the binary size.
```

File: docs/index.rst [hunk 4]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+
+Multi-platform.
+    The OS dependencies are treated as pluggable modules.
+    The user can implement easily his own platform modules in order to use *Micro XRCE-DDS Client* library in his specific platform.
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
    The user can easily implement his platform modules to *Micro XRCE-DDS Client* library in his specific platform.
```

File: docs/index.rst [hunk 5]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+    By default the proyect can run over `Linux`, `Windows` and `Nuttx`.
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
    By default, the project can run over `Linux`, `Windows` and `Nuttx`.
```

File: docs/index.rst [hunk 6]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+
+Free and Open Source.
+   The client library, among with the agent executable, the generator tool and internal dependencies as *MicroCDR* or *FastRTPS* are all of them free and open source.
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
   The client library, the agent executable, the generator tool and internal dependencies as *MicroCDR* or *FastRTPS* are all of them free and open source.
```

File: docs/index.rst [hunk 7]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+
+Easy to use.
+    The project comes with examples of a publisher and a subscriber,
+    an example of how MicroXRCE-DDS is deployed into a stage and
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
    an example of how Micro XRCE-DDS is deployed into a stage and
```

Comment (BorjaOuterelo) [reply docs/index.rst]: I am not sure what do you mean with `stage`.

File: docs/index.rst [hunk 8]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+    an interactive demo that can be used with the `ShapesDemo <https://github.com/eProsima/ShapesDemo>`_ with the purpose of understanding the DDS-XRCE protocol and making tests.
+    The client API es completely explained and documentation comes with a guided example of how create your own client application.
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
    The client API is thoroughly explained, and a guided example of how to create your client application is distributed as part of the documentation.
```

File: docs/index.rst [hunk 9]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+
+Implementation of the DDS-XRCE standard.
+    `DDS-XRCE <https://www.omg.org/spec/DDS-XRCE/1.0/Beta1/PDF>`_ is a standard communication protocol of OMG consortium
+    focused in communicating eXtremely Resource Constrained Environments with the DDS world.
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
    focused on communicating eXtremely Resource Constrained Environments with the DDS world.
```

File: docs/index.rst [hunk 10]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+
+Best effort and reliable communication.
+    *Micro XRCE-DDS* supports both, *best effort* for a fast and light communication and *reliable* when the communication reliability is needed.
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
    *Micro XRCE-DDS* supports both, *best effort* for fast and light communication and *reliable* when the communication reliability is needed.
```

File: docs/index.rst [hunk 11]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+
+Transport layer pluggable.
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
Pluggable tansport layer.
```

File: docs/index.rst [hunk 12]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+    Micro XRCE-DDS is not built over a specific transport as *serial* or *UDP*.
+    It is agnostic about the transport used, and give to the user the posibility of implement easily his own transport.
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
    It is agnostic about the transport used, and give the user the possibility of implement easily his tailored transport.
```

File: docs/index.rst [hunk 13]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+    By the fault, *UPD*, *TCP*, and *serial* transports are provided.
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
    Default *UPD*, *TCP*, and *serial* transports are provided.
```

File: docs/index.rst [hunk 14]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+
+Commercial Support.
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
Commercial support available @ `info@eprosima.com`
```

File: docs/index.rst [hunk 15]
```diff
@@ -10,13 +10,53 @@ eProsima Micro XRCE-DDS
...
+    Available: `info@eprosima.com`
```
Comment (BorjaOuterelo) [inline docs/index.rst]:
```suggestion
```

File: docs/index.rst (hunk 12 above)
Comment (lemunozm) [reply docs/index.rst]: I realised that maybe is algo `...the possibility of implementing...` ?

File: docs/index.rst (hunk 15 above)
Comment (lemunozm) [reply docs/index.rst]: I need this line, because the rst style request me a new line over the "feature title"

File: docs/index.rst (hunk 7 above)
Comment (lemunozm) [reply docs/index.rst]: Into a "real scenario"

File: docs/index.rst (hunk 12 above)
Comment (BorjaOuterelo) [reply docs/index.rst]: Maybe is better `the chance of implementing...`

File: docs/index.rst (hunk 15 above)
Comment (BorjaOuterelo) [reply docs/index.rst]: Need to change the email to point to `support@eprosima.com` then.

**Pull request merged**